### DISCUSSION TITLE
hideNextButton prop

### DISCUSSION THREAD

**Discussion by davidshumway** [gilbarbara/react-joyride#1084] [Ideas]
There may be cases where it's preferred that the user move the tour ahead by performing some action, such as clicking a hamburger menu. But it may also be desirable that the user can go back to the previous step and/or skip the tour. In this case, there could be a prop to hide the next button. A workaround at the moment to use styles:

```javascript
styles: {
  buttonNext: { // no prop to hide next!
    display: 'none'
  }
}
```

**Comment by gilbarbara** [comment]
Hey @davidshumway

If you need additional customization, you can use a custom tooltip component.
